metricbeat/module/redis/mtest/runner.go

```diff
@@ -0,0 +1,36 @@
+package mtest
+
+import (
+	"github.com/elastic/beats/libbeat/tests/compose"
+)
+
+var (
+	Runner = compose.TestRunner{
+		Service: "redis",
+		Options: map[string][]string{
```

Comment:
This is great. Perhaps in the future this could even be generated from the `fields.yml` file.

Reply:
Yes, I want to keep this as a struct with basic data in case we move it to a yaml file.